tests/installation/bootloader_svirt.pm

```diff
@@ -120,16 +119,16 @@ sub run() {
     $svirt->add_vnc({port => '5901'});
 
     if ($vmm_family eq 'kvm') {
-        $svirt->add_interface({type => 'network', source => {network => 'default'}, model => {type => 'virtio'}});
+        $svirt->add_interface({type => 'bridge', source => {bridge => 'br0'}, model => {type => 'virtio'}});
     }
     elsif ($vmm_family eq 'xen') {
         if ($vmm_type eq 'hvm') {
-            $svirt->add_interface({type => 'network', source => {network => 'default'}, model => {type => 'netfront'}});
+            $svirt->add_interface({type => 'bridge', source => {bridge => 'br0'}, model => {type => 'netfront'}});
             # emulator is not being set for Xen HVM automatically
             $svirt->add_emulator({emulator => '/usr/lib/xen/bin/qemu-system-i386'});
         }
         elsif ($vmm_type eq 'linux') {
-            $svirt->add_interface({type => 'network', source => {network => 'default'}});
+            $svirt->add_interface({type => 'bridge', source => {bridge => 'br0'}});
         }
```

Comment:
I could do that if `Net::SSH2` supported retrieving output from executed command. Strangely it seems it doesn't. `Net::OpenSSH` supports that but I am not sure we want to use two SSH modules.. Another option is to rewrite svirt from `Net::SSH2` to `Net::OpenSSH` but I guess @coolo had reason to choose `Net::SSH2`.